packages/cadl-ranch-specs/http/special-headers/repeatability/main.tsp

```diff
@@ -0,0 +1,28 @@
+import "@typespec/http";
+import "@typespec/versioning";
+import "@azure-tools/cadl-ranch-expect";
+
+using TypeSpec.Http;
+using TypeSpec.Versioning;
+
+@doc("Illustrates OASIS repeatability headers")
+@supportedBy("dpg")
+@scenarioService("/special-headers/repeatability")
+namespace SpecialHeaders.Repeatability;
+
+@scenario
+@scenarioDoc("""
+Check we recognize Repeatability-Request-ID and Repeatability-First-Sent.
+""")
+@doc("""
+Check we recognize Repeatability-Request-ID and Repeatability-First-Sent.
+""")
+@post
```

Comment:
shall we also cover put and delete?

Reply:
It should be no difference for the generated code. Since the test for repeatable behavior should not be the goal of cadl-ranch, I just add an operation to azure traits and rename this operation.